internal/skr/client.go

```diff
@@ -0,0 +1,92 @@
+package skr
+
+import (
+	"context"
+	"fmt"
+	"github.com/kyma-project/eventing-auth-manager/internal/ias"
+	v1 "k8s.io/api/core/v1"
+	apiErrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/tools/clientcmd"
+	ctrlclient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	applicationSecretName      = "eventing-webhook-auth"
```

Comment:
The name "eventing-webhook-auth" is only re-used in tests and the Readme. I usually want to avoid exporting code for test only. Also this string is part of the contract between this application and an application consuming this secret. By allowing the test to automatically adjust when the production code is changed, we somehow eliminate the possibility of changes becoming visible during testing.

Reply:
I understand what you mean from theoretical point of view. However, if you change in the business logic, tests will fail which means tests are wrong. If you used reusable constant, tests doesn't fail and it'd be convenient in my opinion. This is just a nitpick.